team/team.2016.md

```diff
@@ -0,0 +1,63 @@
+{% include team-member.html name="Aziz Ketari" caption="Engine" img="2018/aziz-ketari.jpg" %}
+{% include team-member.html name="Cesar Trujillo" caption="Engine" img="2018/cesar-trujillo.jpg" %}
+{% include team-member.html name="Chad Tarpley" caption="Pedal Box" img="2018/chad-tarpley.jpg" %}
+{% include team-member.html name="Emanuil Yanev" caption="Body" img="2018/emanuil-yanev.jpg" %}
+{% include team-member.html name="Jason Liang" caption="Tires" img="2018/jason-liang.jpg" %}
+{% include team-member.html name="Joe Wihbey" caption="Brakes" img="2018/joe-wihbey.jpg" %}
+{% include team-member.html name="Julia Di" caption="Public Relations" img="2018/julia-di.jpg" %}
+{% include team-member.html name="Karim Abdallah" caption="Sensors" img="2018/karim-abdallah.jpg" %}
+{% include team-member.html name="Kristina Andreyeva" caption="Exhaust" img="2017/kristina-andreyeva.jpg" %}
+{% include team-member.html name="Oliver Gruterich" caption="Engine" img="2018/oliver-gruterich.jpg" %}
+{% include team-member.html name="Patrick Naughton" caption="Suspension" img="2018/patrick-naughton.jpg" %}
+{% include team-member.html name="Alex Freemantle" caption="IT" img="2017/alex-freemantle.jpg" %}
+{% include team-member.html name="Peter Richards" caption="IT" img="2018/peter-richards.jpg" %}
+{% include team-member.html name="Ricardo Fritzke" caption="Engine" img="2018/ricardo-fritzke.jpg" %}
+{% include team-member.html name="Tamas Sarvary" caption="Suspension" img="2018/tamas-sarvary.jpg" %}
+{% include team-member.html name="Trevor Docter" caption="Engine" img="2018/trevor-docter.jpg" %}
+{% include team-member.html name="Xuelong Mu" caption="Fuel" img="2016/xuelong-mu.jpg" %}
+{% include team-member.html name="Anna Juchnicki" caption="Suspension" img="2017/anna-juchnicki.jpg" %}
+{% include team-member.html name="Ryan Brenner" caption="Exhaust" img="2016/ryan-brenner.jpg" %}
+{% include team-member.html name="Bruce Lee" caption="Intake" img="2015/bruce-lee.jpg" %}
+{% include team-member.html name="Alex Malouf" caption="Marketing & Brand" img="2018/alex-malouf.jpg" %}
+{% include team-member.html name="Manjul Pahwa" caption="Brakes" img="2018/manjul-pahwa.jpg" %}
+{% include team-member.html name="Tom Rasmussen" caption="Hubs & Uprights" img="2017/tom-rasmussen.jpg" %}
+{% include team-member.html name="Matt Sheridan" caption="Suspension" img="2016/matt-sheridan.jpg" %}
+{% include team-member.html name="Christine Sison" caption="Brakes" img="2018/christine-sison.jpg" %}
+{% include team-member.html name="Diego Song" caption="Electric Racecar" img="2016/diego-song.jpg" %}
+{% include team-member.html name="Pierre Suignard" caption="Sponsorship" img="2018/pierre-suignard.jpg" %}
+{% include team-member.html name="Nicho Villalobos" caption="Aerodynamics" img="2018/nicho-villalobos.jpg" %}
+{% include team-member.html name="Yumou “William” Wei" caption="Body & Steering" img="2017/yumou-wei.jpg" %}
```

Comment:
Ahhh, I didn't notice the ordering at first. I will order by last name. Should I reset head back to before I replaced existing photos and then make the changes you have requested above?